test/utils.ts

```diff
@@ -6,7 +6,7 @@ export interface SignerWithAddress extends Signer {
   address: string;
 }
 
-export async function getContractFactory(name: string, signer: SignerWithAddress) {
+export async function getContractFactory(name: string, signer: Signer) {
```

Comment:
I made these normal signers because it seemed like there was nothing in these methods that explicitly _required_ them to be `SignerWithAddress`.